src/translation/english.c

```diff
@@ -71,6 +72,22 @@ static translation_string all_strings[] = {
     {TR_HOTKEY_CYCLE_LEGION, "Cycle through legions"},
     {TR_HOTKEY_ROTATE_MAP_LEFT, "Rotate map left"},
     {TR_HOTKEY_ROTATE_MAP_RIGHT, "Rotate map right"},
+    {TR_HOTKEY_BUILD_CLEAR_LAND, "Clear land tool"},
+    {TR_HOTKEY_BUILD_VACANT_HOUSE, "Build house"},
+    {TR_HOTKEY_BUILD_ROAD, "Build road"},
+    {TR_HOTKEY_BUILD_PLAZA, "Build plaza"},
+    {TR_HOTKEY_BUILD_GARDENS, "Build gardens"},
+    {TR_HOTKEY_BUILD_PREFECTURE, "Build prefecture"},
+    {TR_HOTKEY_BUILD_ENGINEERS_POST, "Build engineers' post"},
+    {TR_HOTKEY_BUILD_DOCTOR, "Build doctor"},
+    {TR_HOTKEY_BUILD_GRANARY, "Build granary"},
+    {TR_HOTKEY_BUILD_WAREHOUSE, "Build warehouse"},
+    {TR_HOTKEY_BUILD_MARKET, "Build market"},
+    {TR_HOTKEY_BUILD_WALL, "Build wall"},
+    {TR_HOTKEY_BUILD_GATEHOUSE, "Build gatehouse"},
+    {TR_HOTKEY_BUILD_RESERVOIR, "Build reservoir"},
+    {TR_HOTKEY_BUILD_AQUADUCT, "Build aquaduct"},
+    {TR_HOTKEY_BUILD_FOUNTAIN, "Build fountain"},
```

Comment:
I was thinking, we already have the building names in the language files. If we say just "Prefecture" instead of "Build prefecture", we don't need translations into 11 languages for these. (it doesn't have to change in this PR, just something to consider for later)

Reply:
That's a great point, and I think it's not confusing since it's under a clear header. The header isn't translated though.

Reply:
FYI: I just implemented this in 21b48c183eb2a9d6a72c9f5c474e3b5a26d87397.
We can request translations for the header once the feature is stable.